interpreter/container_mutation_test.go

```diff
@@ -34,6 +34,44 @@ import (
 	. "github.com/onflow/cadence/test_utils/sema_utils"
 )
 
+// native helpers
+func assertHelloLog(t *testing.T, invoked *bool) stdlib.StandardLibraryValue {
```

Comment:
Maybe improve the naming: The functions do not `assert` immediately, they create and return a new value, so maybe name them something like "`newAssertHelloLogFunction`"